misk/src/main/kotlin/misk/web/jetty/WebActionsServlet.kt

```diff
@@ -82,7 +86,7 @@ internal class WebActionsServlet @Inject constructor(
       val candidateActions = boundActions.mapNotNull {
         it.match(httpCall.dispatchMechanism, requestContentType, requestAccepts, httpCall.url)
       }
-      val bestAction = candidateActions.sorted().firstOrNull()
+      val bestAction = candidateActions.min()
```

Comment:
I don't know what's our stance on minor, somewhat-unrelated, cosmetic changes (no contribution guidelines) so I threw this in (separate commit). Happy to drop/yank into separate PR.

Reply:
Tidying up along the way is nice.